src/components/TodoContainer.js

```diff
@@ -1,37 +1,99 @@
 import React from 'react';
+import { v4 as uuidv4 } from 'uuid';
+import Header from './Header';
+import InputTodo from './InputTodo';
+import TodoList from './TodoList';
 
 class TodoListContainer extends React.Component {
   constructor(props) {
     super(props);
     this.state = {
       todo: [
         {
-          id: 1,
+          id: uuidv4(),
           title: 'setup dev environement',
-          completed: true,
+          completed: false,
         },
         {
-          id: 2,
-          title: 'lov is nothing for a programmer',
-          completed: true,
+          id: uuidv4(),
+          title: 'No balance nothing for a programmer',
+          completed: false,
         },
         {
-          id: 3,
-          title: 'Let think positif',
-          completed: true,
+          id: uuidv4(),
+          title: 'Suceed to render one stage',
+          completed: false,
         },
       ],
     };
   }
 
+  // using prevState methode to toggle the checkbox
+  // handleChange = (id) => {
+  //   this.setState((prevState) => ({
+  //     todo: prevState.todo.map((item) => {
+  //       if (item.id === id) {
+  //         return {
+
+  //           ...item, completed: !item.completed,
+  //         };
+  //       }
+  //       return item;
+  //     }),
+  //   }));
+  // };
+
```

Comment:
Remember to delete commented code once your finished in order to make your code cleaner 👍